packages/utils/src/miscellaneous.ts

```diff
@@ -0,0 +1,26 @@
+const TAG = 'utils/src/miscellaneous'
```

Comment:
Do you have any other name ideas for this file? `async-helpers`?

Reply:
+1 for rename here